org.kde.akregator.json

```
@@ -0,0 +1,495 @@
+            "name": "kmailtransport",
+            "buildsystem": "cmake-ninja",
+            "config-opts": [
+                "-DBUILD_TESTING=OFF",
+                "-DCMAKE_BUILD_TYPE=RelWithDebInfo"
+            ],
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://download.kde.org/stable/release-service/21.12.1/src/kmailtransport-21.12.1.tar.xz",
+                    "sha256": "a2ada2ecf6532cd347b8cc630d6ba1429b02402cc6f95a3a4b72d1dd012ecf56"
+                }
+            ]
+        },
+        {
+            "name": "kmbox",
+            "config-opts": [
+                "-DBUILD_TESTING=OFF",
+                "-DCMAKE_BUILD_TYPE=RelWithDebInfo"
+            ],
+            "buildsystem": "cmake-ninja",
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://download.kde.org/stable/release-service/21.12.1/src/kmbox-21.12.1.tar.xz",
+                    "sha256": "8aa4e44ba2713e4b70cc8d6b79fb11057e1839f451196b21452c7e50c3e2bcb4"
+                }
+            ]
+        },
+        {
+            "name": "messagelib",
+            "buildsystem": "cmake-ninja",
+            "config-opts": [
+                "-DBUILD_TESTING=OFF",
+                "-DCMAKE_BUILD_TYPE=RelWithDebInfo"
+            ],
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://download.kde.org/stable/release-service/21.12.1/src/messagelib-21.12.1.tar.xz",
+                    "sha256": "ac2d4bf0d6ee66efb604bcb70a346b3762570a34666764662b7dc5c03bccdcab"
+                }
+            ],
+            "modules": [
+                {
+                    "name": "qca",
+                    "config-opts": [
+                        "-DENABLE_TESTING=OFF"
+                    ],
+                    "buildsystem": "cmake-ninja",
+                    "sources": [
+                        {
+                            "type": "archive",
+                            "url": "https://download.kde.org/stable/qca/2.3.4/qca-2.3.4.tar.xz",
+                            "sha256": "6b695881a7e3fd95f73aaee6eaeab96f6ad17e515e9c2b3d4b3272d7862ff5c4"
+                        }
+                    ]
+                }
+            ]
+        },
+        {
+            "name": "kuserfeedback",
+            "config-opts": [
+                "-DBUILD_TESTING=OFF",
+                "-DCMAKE_BUILD_TYPE=RelWithDebInfo"
+            ],
+            "buildsystem": "cmake-ninja",
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://download.kde.org/stable/kuserfeedback/kuserfeedback-1.0.0.tar.xz",
+                    "sha256": "5a2f53ebb4b99a280757ca32bd9b686a7764a726e7e4d8bafee33acbb44b9db7"
+                }
+            ]
+        },
+        {
+            "name": "akregator",
+            "buildsystem": "cmake-ninja",
```

```suggestion
            "buildsystem": "cmake-ninja",
            "post-install": [
                "mv ${FLATPAK_DEST}/share/mime/packages/akonadi-mime.xml ${FLATPAK_DEST}/share/mime/packages/${FLATPAK_ID}.xml"
            ],
```

This export the MIME file properly